x-pack/plugins/security_solution/public/management/components/endpoint_responder/status_action.tsx

```diff
@@ -127,50 +127,83 @@ export const EndpointStatusActionResult = memo<
     return <ResultComponent showAs="pending" />;
   }
 
-  return (
-    <ResultComponent showTitle={false}>
-      <EuiFlexGroup wrap={false} responsive={false}>
-        <EuiFlexItem grow={false}>
-          <EuiText size="s">
-            <FormattedMessage
-              id="xpack.securitySolution.endpointResponseActions.status.agentStatus"
-              defaultMessage="Agent status"
-            />
-          </EuiText>
-          <EndpointAgentAndIsolationStatus
-            status={endpointDetails.host_status}
-            isIsolated={Boolean(endpointDetails.metadata.Endpoint.state?.isolation)}
-            {...pendingIsolationActions}
-          />
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiText size="s">
-            <FormattedMessage
-              id="xpack.securitySolution.endpointResponseActions.status.version"
-              defaultMessage="Version"
-            />
-          </EuiText>
-          <EuiText>{endpointDetails.metadata.agent.version}</EuiText>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiText size="s">
-            <FormattedMessage
-              id="xpack.securitySolution.endpointResponseActions.status.policyStatus"
-              defaultMessage="Policy status"
-            />
-          </EuiText>
-          <EndpointAppliedPolicyStatus
-            policyApplied={endpointDetails.metadata.Endpoint.policy.applied}
-          />
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiText size="s">
-            <FormattedMessage
-              id="xpack.securitySolution.endpointResponseActions.status.lastActive"
-              defaultMessage="Last active"
-            />
-          </EuiText>
-          <EuiText>
+  const statusDescriptionList = () => {
```

Comment:
I would suggest you move this inside a `useCallback` and change the name by something like `getStatusDescriptionList`